packages/pacstall-gui-git/pacstall-gui-git.pacscript

```diff
@@ -23,8 +23,10 @@ install() {
   sudo install -Dm755 pacstall-gui "${STOWDIR}/${name}/usr/bin/pacstall-gui"
   sudo mv -f ./settings.sh ./disable-root-check.diff "${STOWDIR}/${name}/usr/lib/pacstall-gui"
   sudo install -Dm755 LICENSE "${STOWDIR}/${name}/usr/share/licenses/pacstall-gui/LICENSE"
+  sudo mv -f ./pacstall-gui.desktop "${STOWDIR}/${name}/usr/share/applications"
```

Comment:
The reason this is failing is due to the fact that the applications folder is missing and mv doesn't make the folder